projects/angular4-paystack/src/lib/angular4-paystack.component.ts

```diff
@@ -9,7 +9,8 @@ declare var window: MyWindow;
 
 @Component({
   selector: 'angular4-paystack',
-  template: `<button [ngClass]="class" [ngStyle]="style" (click)="pay()"><ng-content></ng-content></button>`,
+  template: `<button [class]="class" [style]="style" (click)="pay()"><ng-content></ng-content></button>`,
```

Comment:
Same as above

Reply:
works as expected on angular 8, 16 and 17